frontend/src/app/features/bim/ifc_models/pages/viewer/ifc-viewer-page.component.ts

```diff
@@ -58,6 +59,10 @@ export class IFCViewerPageComponent extends PartitionedQuerySpacePageComponent {
         allowed: ['work_packages.createWorkPackage', 'work_package.copy'],
       },
     },
+    {
+      component: RefreshButtonComponent,
+      show: () => window.navigator.userAgent.search('Revit') > -1,
```

Comment:
You should probably use the `ViewerBridgeService` which already has this logic defined to check whether the button is used.